system-int-test/test/cmr/system_int_test/search/variable/force_delete_test.clj

```diff
@@ -4,85 +4,110 @@
+                                                   :Name "Variable1"
+                                                   :LongName "LongName3"
+                                                   :provider-id "PROV2"})]
+    (index/wait-until-indexed)
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;;; Tests
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+    (testing "force delete variable revision"
+      ;; All variable revisions exist before force deletion
+      (variable/assert-variable-references-match
+       [var1-1 var1-2-tombstone var1-3 var2-1 var2-2-tombstone var2-3 var3]
+       (search/find-refs :variable {:all-revisions true}))
+      ;; force delete a regular revision
+      (mdb/force-delete-concept
+        (:concept-id var1-1) (:revision-id var1-1))
+      ;; force delete a tombstone revision
+      (mdb/force-delete-concept
+        (:concept-id var2-2-tombstone) (:revision-id var2-2-tombstone))
+      (index/wait-until-indexed)
+      ;; force deleted revisions are no longer in the search result
+      (variable/assert-variable-references-match
+       [var1-2-tombstone var1-3 var2-1 var2-3 var3]
+       (search/find-refs :variable {:all-revisions true}))
+      ;; Associate a collection and variable
+      (au/associate-by-concept-ids token
+                                   (:concept-id var1-3)
+                                   [{:concept-id (:concept-id coll1)}])
+      (index/wait-until-indexed)
+      (let [expected-associations {:collections [{:concept-id (:concept-id coll1)}]}]
+        (testing "associations to collections are captured in the variables all revisions endpoint"
+          (variable/assert-variable-associations var1-3 expected-associations
+                                                 {:all_revisions true}))
+        (let [var1-4 (variable/ingest-variable var1-concept)]
+          (index/wait-until-indexed)
+          (testing (str "associations are correct in the all revisions endpoint when ingesting a "
+                        "new variable revision")
+            (variable/assert-variable-associations var1-4 expected-associations
+                                                   {:all_revisions true}))
+          (testing "force delete does not cascade to variable association"
+            ;; search collections by variable native-id found the collection
+            (d/refs-match? [coll1] (search/find-refs :collection {:variable_native_id "var1"}))
+            ;; force delete the latest revision of the variable cascade to variable association
+            (mdb/force-delete-concept (:concept-id var1-3) (:revision-id var1-3))
+            (index/wait-until-indexed)
+            ;; search collections by variable native-id no longer found the collection
+            (d/refs-match? [coll1] (search/find-refs :collection {:variable_native_id "var1"}))))))
 
-(deftest force-delete-variable-with-associations
```

Comment:
/me sighs ... I'll miss that clean test set-up ;-)  (I feel that we do too much in each of our tests ... they become hard to maintain and adjust, easy to accumulate cruft as things in the CMR change ...)

That being said, I can see that it's easier just to use the old one than update the new one for the needed additions.